README.md

```diff
@@ -128,8 +128,11 @@ The following features are available.
 
 *  `comments` - allow commands (see below) in comments.
 
+*  `approvers` - approvers are allowed to trigger merge into the repository.
```

Comment:
approvers are allowed to merge pull requests.

Reply:
ok